Equality/Views/Teams/TeamPage.xaml

```diff
@@ -15,6 +15,26 @@
            d:Background="{StaticResource MaterialDesignPaper}"
            d:DataContext="{d:DesignInstance vm:TeamPageViewModel, IsDesignTimeCreatable=True}"
            mc:Ignorable="d">
+    <Page.Resources>
+        <Style BasedOn="{StaticResource MaterialDesignTabRadioButton}"
```

Comment:
I believe you define this style in 2+ places, so move this in resource dictionary for all application (`Themes/Buttons`)